### PR TITLE
Fix redundat separator in ride's measurement & test data window tab

### PR DESCRIPTION
**Pull request by octaroot**
This fixes issue #3647.

The file `src/windows/ride.c` should really be marked for refactoring, strings are referenced via hard-coded id's as opposed to `string_ids.h` constants.

### CODE REVIEW

**Comment by IntelOrca** [comment]
@octaroot there are a lot of hard coded string IDs all over the place, not just ride.c

File: src/windows/ride.c [hunk 1]
```diff
@@ -4992,17 +4996,21 @@ static void window_ride_measurements_paint(rct_window *w, rct_drawpixelinfo *dpi
...
 				}
 				if (numLengths == 0) {
-					set_format_arg(0, uint16, 1346);
+					set_format_arg(0, uint16, STR_RIDE_LENGTH_ENTRY);
 					set_format_arg(2, uint16, 0);
 					numLengths++;
+				} else {
+					//sadly, STR_RIDE_LENGTH_ENTRY_WITH_SEPARATOR are defined with the separator AFTER an entry
+					//therefore we set the last entry to use the no-separator format now, post-format
+					set_format_arg(0 + ((numTimes - 1) * 4), uint16, STR_RIDE_LENGTH_ENTRY);
```
Comment (janisozaur) [inline src/windows/ride.c]: shouldn't it be `numLengths` in here?

Comment (octaroot) [inline src/windows/ride.c]: It definitely should. Thanks for that. I already commited a fix and pushed to the branch used in this PR .. but I guess due to this PR being merged already, it won't show up here. Should I open a new PR then?

Comment (janisozaur) [inline src/windows/ride.c]: yes

**Pull request merged**